### PR TITLE
Added first run of casepro contact sync

### PR DESCRIPTION
**Pull request by jpaidoussi**

### CODE REVIEW

File: migrationtools/migrations.py [hunk 1]
```diff
@@ -721,6 +723,60 @@ def migrate_subscriptions(self, active_only=True, start=None, stop=None, limit=N
...
+                for value in scheme_addresses:
+                    if scheme == "msisdn":
+                        scheme = "tel"
+                    urns.append("%s:%s" % (scheme, value))
+
+            contact_details = {
+                'uuid': uuid,
+                'language': lang,
+                'org_id': 1,  # hard default to 1
+                'name': None,
```
Comment (rudigiesler) [inline migrationtools/migrations.py:761]: Do we not have names in the identity store for momconnect?

**Review by rudigiesler — COMMENTED** [review]
I'm not sure about the sqlalchemy things, never used it before. Just a question about the name field, otherwise looks good to me.

File: migrationtools/migrations.py (hunk 1 above)
Comment (jpaidoussi) [reply migrationtools/migrations.py:761]: Nope

**Review by rudigiesler — APPROVED** [review]
:+1:

File: migrationtools/databases.py [hunk 2]
```diff
@@ -279,7 +279,7 @@ def update_identity(self, uid, identity_data):
         return self.execute(statement)
 
     def get_all_identities(self):
-        statement = select([self.identity]).order_by(self.identity.c.created_at)
+        statement = select([self.identity]).order_by(self.identity.c.created_at).limit(10)
```
Comment (rudigiesler) [inline migrationtools/databases.py]: I'm a bit confused by the limit? Does this batch them, or does it only fetch 10?

**Comment by rudigiesler** [comment]
:+1:

File: migrationtools/databases.py (hunk 2 above)
Comment (jpaidoussi) [reply migrationtools/databases.py]: That was there for testing, it was removed in commit https://github.com/praekeltfoundation/momconnect-migration-tools/pull/13/commits/5fbe9389b4b7e013d396146755093c55f388d02f

**Pull request merged**